releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,18 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 5.2.0 - November 29, 2022
+* Added support for .NET 7 as a target framework. [#366](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/366)
+* Fixed a bug where passing an empty string for the label filter to `AzureAppConfigurationOptions.Select` would select key-values with all labels. [#311](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/311)
+* Fixed a bug where calls to `AzureAppConfigurationOptions.Select` would be incorrectly ignored if the key-values were already selected by a previous call. [#349](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/349)
+* Fixed a bug where `AddAzureAppConfiguration` could throw an exception on invalid feature flags even with the `optional` parameter set to `true`. [#348](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/348)
+* Upgraded the package references `Azure.Security.KeyVault.Secrets` to `4.3.0` and `System.Text.Json` to `4.7.2`. [#351](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/351)
+* Removed the following new API for the App Configuration geo-replication support. [#178](https://github.com/Azure/AppConfiguration/issues/178)
```

Comment:
While this is true compared with the last preview release, I'm a little concerned about how this would be perceived by customers :). We are not implying we are removing the geo-replication support, right?

It's really about what it compares to. The geo-replication API was never made in a stable release. I think we can drop it from the release nodes. This applies to the v2 schema too.

We didn't follow it strictly in the past, but it's not too late to start it now. For a stable version release, we should include all items that are new compared to the previous stable version, even if some of them have been mentioned in a previous preview release. It's good to call them out if a feature in a preview made it into a stable release.